website/src/theme/Footer/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
Why do to swizzle Footer if nothing new has been added to it? It does not make sense.

Reply:
Due to documentation and theme

Reply:
@lex111 is right. Don't need to swizzle it. Just use the default.

Reply:
Not fixed it.